### DISCUSSION TITLE
Why my device can't handle 80 frames animation?

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#174] [Q&A] [answered]
I tried to post some animation. First animation consists of around 40 frames and second one consists of around 50 . But none of the animation works fine. Why?

**Comment by zdunex25** [comment]
I know you post a lot here, but you should mention device model anyway. ZeppOS 1.0 device has more limitations for example.

**Comment by MyoungXUE** [comment]
Hello, I received your feedback, I will provide a complete animation example to your reference as soon as possible.

**Comment by MyoungXUE** [comment, marked as answer]
Hi, I have added a complete `IMG_ANIM` reference example, hope it can help you

https://github.com/zepp-health/zeppos-samples/blob/main/application/2.0/showcase/page/ui/widget/img_anim.js

**Reply by alexmathew11** [reply]
Thanks man It worked but still I am finding an issue . When the animation finishes black screen comes for 1 or 0.5 seconds . Why is that? Can you help me
